pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -1392,6 +1392,15 @@ func ServiceLabels(name string) map[string]string {
 	}
 }
 
+func ConfigureCloudEvent(m *infinispanv1.Infinispan, c *config.InfinispanConfiguration) {
```

Comment:
I think it's better to place this logic to the config controller rather than overload infinispan_controller with unspecific code

Reply:
Ok by me.
I've just place it in the same place as `ConfigureServerEncryption`, better to move both?

Reply:
Yes, sure, thanks @rigazilla
It would be nice to places all configuration related code to the config_controller if possible

Reply:
`ConfigureServerEncryption` is also used by the zero_controller, I'm leaving it where it is